doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java

```diff
@@ -432,7 +432,7 @@ public void renderDocument( Writer writer, RenderingContext docRenderingContext,
             }
             sink.enableLogging( new PlexusLoggerWrapper( getLogger() ) );
 
-            doxia.parse( reader, docRenderingContext.getParserId(), sink );
+            doxia.parse( reader, docRenderingContext.getParserId(), sink , docRenderingContext.getInputName() );
```

Comment:
Remove the space before the comma

Reply:
Done! Thanks a lot for your dedication :bow: